docs/docs/workflows/create.md

```diff
@@ -252,14 +252,13 @@ management tooling of your choice. You need to keep the essential functionality
 :::info
 
   On Azure, if the enforcement policy is set to `MAAFallback` in `constellation-config.yaml`, a manual update to the MAA provider's policy is necessary.
-  You can apply the update with the following commands, where `<VERSION>` is the version of Constellation that should be set up. (e.g. `v2.12.0`)
+  <!-- vale off -->
+  You can apply the update with the following command after creating the infrastructure, with `<URL>` being the URL of the MAA provider. (i.e. `$(terraform output attestationURL | jq -r)`, when using the minimal Terraform configuration.)
 
   ```bash
-  git clone --branch <VERSION> https://github.com/edgelesssys/constellation
-  cd constellation/hack/maa-patch
-  go run . $(terraform output attestationURL | jq -r)
+  constellation maa-patch <URL>
   ```
-
+   <!-- vale on -->
```

Comment:
nit: this can be above the code block

Reply:
It cannot be inside the info block unfortunately, see the [build logs](https://app.netlify.com/sites/constellation-docs/deploys/65523a73195c900009569b53)